.github/workflows/ci.yaml

```
@@ -77,6 +77,9 @@ jobs:
       - name: Checking consistency between json and proto files
         run: task test:proto
 
+      - name: Server code compilation and tests
```

maybe a better name? similar to `OASF schema validation`

```suggestion
      - name: OASF Server Validation
```